### ISSUE TITLE
Safe zone (Not to avoid triggers, but for ressources)

### ISSUE THREAD

**Issue by Alienmen1** [maximo1984/Kingdom-Death-Maximo-Kijan-Mod#127]
There should be a marked location where you can drop safely the ressources your got during the showdown on the survivor's grid unless the survivor's portrait is that said location?

**Comment by maximo1984** [comment]
Agree. I have made a designated spot on the new gear models for this.
